wick/expression.py

```diff
@@ -839,13 +863,24 @@ def _print_str(self):
             out += sign + str(num) + t._print_str(with_scalar=False) + "\n"
         return out[:-1]
 
-    def _print_einsum(self, lhs=None):
+    def _print_einsum(self, lhs=None, exprs_with_space = None, names_with_space = None, space_idx_formatter = None, einsum_str_formatter = None):
```

Comment:
The line is too long and there shouldn't be spaces around the `=` in the function definition